### PR TITLE
Lööps!

### PR DESCRIPTION
**Pull request by 89netraM**

### CODE REVIEW

File: src/loops.rs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	body: impl FnMut(C::Item) -> M + 'm,
+) -> NextM
+where
+	C: IntoIterator,
+	C::IntoIter: Clone + 'm,
+	M: Monad<'m, LoopControl, Bind<LoopControl> = NextM>,
+	NextM: Monad<'m, LoopControl>,
+{
+	let iterator = container.into_iter();
+	recurse_for_loop(iterator, body)
```
Comment (Deltaphish) [inline src/loops.rs:14]: This returns `M LoopControl` , change this to `M ()` so LoopControl does not leak out

File: examples/logging.rs
Comment (Deltaphish) [inline examples/logging.rs:1]: Use refrences or RC + clone for mutability in loop.

**Review by Deltaphish — CHANGES_REQUESTED** [review]
Good first attempt, i am verry proud of u 🦖

File: src/loops.rs (hunk 1 above)
Comment (89netraM) [reply src/loops.rs]: Done!

**Review by Deltaphish — APPROVED** [review]
I approve of this 👏

**Pull request merged**